lib/rubocop/config_loader.rb

```diff
@@ -109,9 +109,9 @@ def add_excludes_from_files(config, config_file)
 
       def default_configuration
         @default_configuration ||= begin
```

Comment:
Does this become inconsistent with our default indentation of blocks on assignment?

Reply:
Yes, it does.

Reply:
To elaborate on this - I was referring to everything except `do/end`, as really `begin`/`end` is just a way to pass several expressions where you normally need to pass only one. In this way I think it's much closer to something like conditionals (e.g. `if`, then to `do`).